int32 message_code
float64 source_x
float64 source_y
float64 destination_x
float64 destination_y
float64 depot_x
float64 depot_y


